[2022-08-18 16:55:03,059] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-18T16:50:00+00:00 [queued]>
[2022-08-18 16:55:03,071] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-18T16:50:00+00:00 [queued]>
[2022-08-18 16:55:03,071] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-18 16:55:03,071] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-18 16:55:03,071] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-18 16:55:03,106] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_finalizados> on 2022-08-18 16:50:00+00:00
[2022-08-18 16:55:03,115] {standard_task_runner.py:52} INFO - Started process 6408 to run task
[2022-08-18 16:55:03,129] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_finalizados', 'scheduled__2022-08-18T16:50:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmpvz2mdl_9', '--error-file', '/tmp/tmp_p6ayx6z']
[2022-08-18 16:55:03,147] {standard_task_runner.py:80} INFO - Job 32: Subtask popula_elastic_finalizados
[2022-08-18 16:55:03,329] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-18T16:50:00+00:00 [running]> on host 1e1070d9e5ae
[2022-08-18 16:55:03,460] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_finalizados
AIRFLOW_CTX_EXECUTION_DATE=2022-08-18T16:50:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-18T16:50:00+00:00
[2022-08-18 16:55:03,475] {base.py:271} INFO - GET http://192.168.2.100:9200/ [status:200 request:0.013s]
[2022-08-18 16:55:07,523] {base.py:271} INFO - HEAD http://192.168.2.100:9200/_template/desk_finalizados [status:200 request:0.001s]
[2022-08-18 16:55:07,819] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2022-08-18 16:55:08,011] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.092s]
[2022-08-18 16:55:08,142] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.097s]
[2022-08-18 16:55:08,209] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.065s]
[2022-08-18 16:55:08,331] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.084s]
[2022-08-18 16:55:08,480] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.066s]
[2022-08-18 16:55:08,581] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.063s]
[2022-08-18 16:55:08,659] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.054s]
[2022-08-18 16:55:08,786] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.065s]
[2022-08-18 16:55:08,841] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.053s]
[2022-08-18 16:55:08,866] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.026s]
[2022-08-18 16:55:08,943] {logging_mixin.py:115} INFO - 0       18-08-2022
1       18-08-2022
2       18-08-2022
3       18-08-2022
4       18-08-2022
           ...    
4744    19-07-2022
4745    19-07-2022
4746    19-07-2022
4747    19-07-2022
4748    19-07-2022
Name: DataFinalizacao, Length: 4749, dtype: object
[2022-08-18 16:55:08,945] {python.py:173} INFO - Done. Returned value was: None
[2022-08-18 16:55:08,960] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_finalizados, execution_date=20220818T165000, start_date=20220818T165503, end_date=20220818T165508
[2022-08-18 16:55:09,020] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-18 16:55:09,062] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
